### PR TITLE
feat: download file

### PR DESCRIPTION
**Pull request by goshander**
just for test and discuss

### CODE REVIEW

**Comment by goshander** [comment]
image

FIY
linux use KDialog package without native ubuntu theme support (Qt theme is not default supported in ubuntu)

**Comment by AlexEbenrode** [comment]
Throws error whtn close file explorer

**Comment by AlexEbenrode** [comment]
> Throws error whtn close file explorer

fixed

**Review by AlexEbenrode — APPROVED** [review]

File: lib/pages/bucket_page/bucket_page.dart [hunk 1]
```diff
@@ -70,7 +77,38 @@ class BucketPage extends StatelessWidget {
...
+
+                          displayInfoBar(context, builder: (context, close) {
+                            return InfoBar(
+                              title: const Text('File downloaded'),
+                              content: Text(object.key!),
+                              action: IconButton(
+                                icon: const Icon(FluentIcons.clear),
+                                onPressed: close,
+                              ),
+                              severity: InfoBarSeverity.success,
```
Comment (AlexEbenrode) [inline lib/pages/bucket_page/bucket_page.dart:109]: image

idk how fix it

**Comment by AlexEbenrode** [comment]
fixed pipeline error

**Comment by goshander** [comment]
:tada: this issue has been resolved in version 1.0.0 :tada:

release is available on [GitHub](<github_release_url>)

**Pull request merged**